common/src/main/kotlin/no/nav/su/se/bakover/common/JmsConfig.kt

```diff
@@ -23,7 +23,7 @@ data class JmsConfig(
                 channel = it.mqChannel
                 queueManager = it.mqQueueManager
                 transportType = WMQConstants.WMQ_CM_CLIENT
-                setBooleanProperty(WMQConstants.USER_AUTHENTICATION_MQCSP, false)
+                setBooleanProperty(WMQConstants.USER_AUTHENTICATION_MQCSP, true)
```

Comment:
Kunne evt testet å bumpe klient-versjonen... frister dog å bare gå for dette hvis det funker.

Reply:
Jepp, dette funker ihvertfall, så kan vi se på å bumpe versjonen i tillegg evt. @hestad hadde litt issues forrige gang han prøvde visstnok.